MFVNeutralino/python/Vertexer_cfi.py

```diff
@@ -35,10 +35,10 @@
                                primary_vertices_src = cms.InputTag('goodOfflinePrimaryVertices'),
                                disregard_event = cms.bool(False),
                                use_tracks = cms.bool(True),
-                               #use_separated_leptons = cms.bool(True), #use lepton tracks -> if they were separated at unpackedcandidates
+                               use_separated_leptons = cms.bool(True), #use lepton tracks -> if they were separated at unpackedcandidates
```

Comment:
Is this only relevant for the displaced lepton analysis? @abbywarden

Reply:
(we may want a real switch somewhere to turn it on/off in the vertexer, depending on which analysis is being done. Sort of like our other switches between btag vs. lepton triggers [vs. displaced lepton too, I hope?])

Reply:
Having gone further through this file, I'm guessing use_displaced_lepton is the switch, somewhere!